### PR TITLE
qmv: add -f do example

### PR DESCRIPTION
**Pull request by marekkowalczyk**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v1.3.0

### CODE REVIEW

File: pages/common/qmv.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Move files, but swap the positions of the source and the target filenames in the editor:
 
 `qmv --option swap {{*.jpg}}`
+
+- Rename all files and folders in the current directory, but show only target filenames in the editor (you can think of it as a kind of simple mode):
+
+`qmv -f do .`
```
Comment (EmilySeville7cfg) [inline pages/common/qmv.md:28]:
```suggestion
`qmv --format=do .`
```

Optional fix: long option can be used.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

**Review by blueskyson — APPROVED** [review]

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Changes were approved by mistake, suggestions were fixed.
Mandatory fixes found, changes requested.

**Comment by waldyrious** [comment]
It seems like this PR was merged to `master`, so it doesn't appear in the `main` tag (I think there's an automatic `main`→`master` sync, but not the other way around. Commit 896abba24 should probably be cherry-picked back to `main` so the content isn't lost, especially when we eventually remove the master branch (#9628).

**Pull request merged**